test/TestMultiTag.cpp

```diff
@@ -171,6 +171,11 @@ void TestMultiTag::testReferences(){
     dt.addReference(da_2);
     CPPUNIT_ASSERT_MESSAGE(counterrmsg.str(), dt.referenceCount() == 2);
 
+    std::stringstream haserrmsg;
+    haserrmsg << "TestMultiTag::testReference: Has method did not work!";
```

Comment:
Couldn't that just be `std::string haserrmsg = "TestMultiTag::testReference: Has method did not work!"; `